src/NServiceBus.RavenDB/Internal/RavenUserInstaller.cs

```diff
@@ -51,7 +52,14 @@ internal static void AddUserToDatabase(string identity, DocumentStore documentSt
         {
             var database = documentStore.DefaultDatabase ?? "<system>";
 
-            logger.InfoFormat(string.Format("Adding user '{0}' to raven. Instance:'{1}', Database:'{2}'.", identity, documentStore.Url, database));
+            var credentials = documentStore.Credentials as NetworkCredential;
+            if (credentials != null && !string.IsNullOrWhiteSpace(credentials.UserName) && !string.IsNullOrWhiteSpace(credentials.Password))
+            {
+                logger.InfoFormat("Skipping adding user '{0}' to RavenDB, because credentials were provided: {1}", identity, credentials);
```

Comment:
I would modify the message to just say:
`Skipping adding user '{0}' to RavenDB, because credentials were provided via connection string`